src/locales/de/common.json

```diff
@@ -0,0 +1,368 @@
+{
+  "apisix": {
+    "dashboard": "APISIX Dashboard",
+    "logo": "APISIX Logo"
+  },
+  "consumerGroups": {
+    "singular": "Konsumenten-Gruppe"
+  },
+  "consumers": {
+    "singular": "Konsument"
+  },
+  "credentials": {
+    "singular": "Anmeldeinformation"
```

Comment:
I'm unsure if we should use `Berechtigungsnachweise` 🤔 

https://de.wikipedia.org/wiki/Berechtigungsnachweis_(Identifikationstechnik)